tribits/core/package_arch/TribitsGlobalMacros.cmake

```diff
@@ -1215,53 +1211,41 @@ endmacro()
 
 # Get the versions of all the git repos
 #
-function(tribits_generate_repo_version_file_string  PROJECT_REPO_VERSION_FILE_STRING_OUT)
+function(tribits_generate_repo_version_file_string  projectRepoVersionFileStrOut)
 
-  set(REPO_VERSION_FILE_STR "")
+  set(projectRepoVersionFileStr "")
 
-  if (${${PROJECT_NAME}_SHOW_GIT_COMMIT_PARENTS})
-    tribits_generate_single_repo_version_string(
-      ${CMAKE_CURRENT_SOURCE_DIR}
-      SINGLE_REPO_VERSION INCLUDE_COMMIT_PARENTS)
-  else()
-    tribits_generate_single_repo_version_string(
-      ${CMAKE_CURRENT_SOURCE_DIR} SINGLE_REPO_VERSION)
-  endif()
-  string(APPEND REPO_VERSION_FILE_STR
+  tribits_generate_single_repo_version_string(
+    ${CMAKE_CURRENT_SOURCE_DIR} singleRepoVersionStr
+    INCLUDE_COMMIT_PARENTS ${${PROJECT_NAME}_SHOW_GIT_COMMIT_PARENTS})
+  string(APPEND projectRepoVersionFileStr
     "*** Base Git Repo: ${PROJECT_NAME}\n"
-    "${SINGLE_REPO_VERSION}\n" )
+    "${singleRepoVersionStr}\n" )
 
-  set(EXTRAREPO_IDX 0)
-  foreach(EXTRA_REPO ${${PROJECT_NAME}_ALL_EXTRA_REPOSITORIES})
-
-    #print_var(EXTRA_REPO)
-    #print_var(EXTRAREPO_IDX)
-    #print_var(${PROJECT_NAME}_ALL_EXTRA_REPOSITORIES_DIRS)
+  set(extraRepoIdx 0)
+  foreach(extraRepo ${${PROJECT_NAME}_ALL_EXTRA_REPOSITORIES})
 
     if (${PROJECT_NAME}_ALL_EXTRA_REPOSITORIES_DIRS)
       # Read from an extra repo file with potentially different dir.
-      list(GET ${PROJECT_NAME}_ALL_EXTRA_REPOSITORIES_DIRS ${EXTRAREPO_IDX}
-        EXTRAREPO_DIR )
+      list(GET ${PROJECT_NAME}_ALL_EXTRA_REPOSITORIES_DIRS ${extraRepoIdx}
+        extraRepoDir )
     else()
        # Not read from extra repo file so dir is same as name
-       set(EXTRAREPO_DIR ${EXTRA_REPO})
+       set(extraRepoDir ${extraRepo})
     endif()
-    #print_var(EXTRAREPO_DIR)
 
     tribits_generate_single_repo_version_string(
-       "${CMAKE_CURRENT_SOURCE_DIR}/${EXTRAREPO_DIR}"
-       SINGLE_REPO_VERSION)
-    string(APPEND REPO_VERSION_FILE_STR
-      "*** Git Repo: ${EXTRAREPO_DIR}\n"
-      "${SINGLE_REPO_VERSION}\n" )
-
-    #print_var(REPO_VERSION_FILE_STR)
+       "${CMAKE_CURRENT_SOURCE_DIR}/${extraRepoDir}" singleRepoVersionStr
+       INCLUDE_COMMIT_PARENTS ${${PROJECT_NAME}_SHOW_GIT_COMMIT_PARENTS})
```

Comment:
@achauphan, note that technically there is no automated test that runs this call with `${PROJECT_NAME}_SHOW_GIT_COMMIT_PARENTS = ON`.  But the lack of this test seems like a low risk given there is just a single argument that changes and everything else is tested.

Reply:
@bartlettroscoe, yeah that's a good point. It is only being testing in a single repo case and not in a multi-repo case. Would modifying the test [tribits_add_advanced_test(CTestDriver_TribitsExMetaProj_clone_default_branch_remote)](https://github.com/TriBITSPub/TriBITS/blob/685c8d5fdb94fe591e4dbd1e0687c41943961461/test/ctest_driver/TribitsExampleMetaProject/CMakeLists.txt#L114C1-L114C85) and enabling `${PROJECT_NAME}_SHOW_GIT_COMMIT_PARENTS = ON` be a reasonable method of covering this?